packages/oauth/src/install-provider-options.ts

```diff
@@ -2,17 +2,95 @@ import { Logger, LogLevel } from '@slack/logger';
 import { WebClientOptions } from '@slack/web-api';
 import { StateStore } from './state-stores';
 import { InstallationStore } from './stores';
+import { InstallURLOptions } from './install-url-options';
 
 export interface InstallProviderOptions {
+
+  /**
+   * Client ID, which can be found at https://api.slack.com/apps
```

Comment:
For this and client secret, perhaps the comment could be "which can be found under the Basic Information section of your application on https://api.slack.com/apps" ?

Reply:
```suggestion
   * Client ID, which can be found under the Basic Information section of your application on https://api.slack.com/apps
```